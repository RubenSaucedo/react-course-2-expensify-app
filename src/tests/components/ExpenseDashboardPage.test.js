import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('Should render de DashboardPage', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});