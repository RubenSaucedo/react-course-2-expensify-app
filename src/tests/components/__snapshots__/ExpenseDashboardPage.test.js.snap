// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render de DashboardPage 1`] = `
<div>
  Expense Dashboard Page
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
